source/config-file/protect-connection-strings.txt

```diff
@@ -0,0 +1,86 @@
+.. _compass-protect-connection-strings:
+.. _hide-connection-string-credentials:
+
+==========================================
+Hide Credentials in Your Connection String 
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+When you paste a connection string into the |compass| connection form, by
+default, |compass-short| shows credentials in plaintext. To hide credentials in 
+the connection string, use the ``protectConnectionStrings`` option. 
+
+About This Task
+---------------
+
+When ``protectConnectionStrings`` is enabled, users cannot perform the following 
+actions: 
+
+- Edit the connection string in the |compass-short| connection form 
```

Comment:
I think that because these are complete sentences, we should put periods at the ends of them.